### PR TITLE
Add some covers annotations - #3671

### PR DESCRIPTION
**Pull request by edditor**
This PR does:

* add some @covers annotations #3671

### CODE REVIEW

File: tests/unit/Util/GetoptTest.php [hunk 1]
```diff
@@ -14,9 +14,14 @@
 
 /**
  * @small
+ * @covers \PHPUnit\Util\Getopt
  */
 final class GetoptTest extends TestCase
 {
+    /**
+     * @covers \PHPUnit\Util\Getopt::getopt
```
Comment (localheinz) [inline tests/unit/Util/GetoptTest.php]: Not sure, perhaps the class-level annotation will suffice?

**Comment by sebastianbergmann** [comment]
This branch has conflicts that must be resolved.

**Review by localheinz — APPROVED** [review]
👍

**Comment by localheinz** [comment]
Thank you, @edditor!

File: tests/unit/Util/GetoptTest.php (hunk 1 above)
Comment (sebastianbergmann) [reply tests/unit/Util/GetoptTest.php]: Yes, please only use class-level annotations.

**Pull request merged**